api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -104,6 +104,14 @@ type HeaderSettings struct {
 	// is encountered. The default action is to reject the request.
 	// +optional
 	WithUnderscoresAction *WithUnderscoresAction `json:"withUnderscoresAction,omitempty"`
+
+	// configure Envoy proxy to forward x-forwarded-client-cert (XFCC) HTTP header
+	// +optional
```

Comment:
thoughts on this alternate API
```
xForwardedClientCert:
  mode: "SanitizeSet"
  set: # By default, some/all keys are set (TBD)
  - Cert
  - Chain
  - .....
```

Reply:
cc @envoyproxy/gateway-maintainers

Reply:
mode: "SanitizeSet" --> mode: "Sanitize" ?

Reply:
+1 on putting these two knobs inside a container since they're related.

Reply:
+1 this is good
1 questions is the list, we can add anything there. is there a way to validate input on list?

Reply:
validation can be solved using a custom type e.g. https://github.com/envoyproxy/gateway/blob/2c4d385fcfb02841483cc36e0292175f60249bbb/api/v1alpha1/envoyproxy_types.go#L138

Reply:
done, adding validation.